src/common/infra/ofga.rs

```diff
@@ -39,9 +39,12 @@ use crate::{
 
 pub async fn init() -> Result<(), anyhow::Error> {
     use o2_enterprise::enterprise::openfga::{
-        authorizer::authz::get_tuple_for_new_index, get_all_init_tuples,
+        authorizer::authz::{get_tuple_for_new_index, open_fga_config},
+        get_all_init_tuples,
     };
 
+    let conf = open_fga_config("").await; // todo
```

Comment:
initialize this here or, more likely, in the caller to the `init` function and pass it in